### DISCUSSION TITLE
[Run issue]: OpenTOPAS/OpenTOPAS_quickStart_Debian.md - Step 8.4

### DISCUSSION THREAD

**Discussion by alex-phy** [OpenTOPAS/OpenTOPAS#26] [Run-Problem] [answered]
### Contact Details

guimaraes.alexf@gmail.com

### What was the issue?

I followed every single step for the first installation of OpenTOPAS and encountered a compiler issue in the command line below:
`cmake ../OpenTOPAS - DCMAKE_INSTALL_PREFIX=../OpenTopas-install`
![Screenshot_24](https://github.com/OpenTOPAS/OpenTOPAS/assets/56261730/52673837-f9e9-4d20-af3d-05f9aa229387)

### TOPAS category

Geometry

### Version

4.0

### What Operative System (OS) are you running on?

Windows (WSL)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by icmeyer** [comment, marked as answer]
Hi, thanks for giving feedback on the compilation tutorial. I'm assuming this is from step 8.4 of the Quick Start for Debian Guide. Copying the code block here for reference:

```
    cd $HOME/Applications/TOPAS
    rm -rf OpenTOPAS-install OpenTOPAS-build
    mkdir OpenTOPAS-{build,install}
    cd OpenTOPAS-build
    export Geant4_DIR=$HOME/Applications/GEANT4/geant4-install
    export GDCM_DIR=$HOME/Applications/GDCM/gdcm-install
    cmake ../OpenTOPAS -DCMAKE_INSTALL_PREFIX=../OpenTOPAS-install
    sudo make -j20 install
```

 Did you properly set the Geant4_DIR environment variable? You can check what is by running `echo $Geant4_DIR`  before running the cmake command. If it is properly set, there may be a problem with your Geant4 installation, you can check the contents of the install with `ls $Geant4_DIR/bin`, and it should have a file like `geant4-config`

If there are no files where your `$Geant4_DIR` is set, you will have to go back to the Geant4 installation steps. Let me know if you run into more troubles!

**Reply by alex-phy** [reply]
Hi, thanks for the guidance. I tried to follow the step-by-step instructions for WSL, but I kept encountering error messages when trying to run the examples. I decided to do a dual boot installation of Ubuntu and successfully installed OpenTOPAS. However, now I'm having issues with the graphical part. When I run QtShapeTest.txt, I don't get any graphical response.
![Captura de tela de 2024-07-14 14-22-17](https://github.com/user-attachments/assets/25867581-c33c-4dd1-8210-820e1208454b)
